test/e2e/README.md

```diff
@@ -24,11 +24,36 @@ knative-gcp should be added under [knative-gcp e2e test lib](lib).
 
 ## Running E2E Tests on an existing cluster
 
-To run [the e2e tests](../e2e) with `go test` command, you need to have a
-running environment that meets
-[the e2e test environment requirements](#environment-requirements), and you need
-to specify the build tag `e2e`.
+### Prerequisites
+There are two ways to set up authentication mechanism. 
+- If you want to run E2E tests with authentication mechanism using **Workload Identity**, please configure the authentication mechanism with **Workload Identity**.
+- If you want to run E2E tests with authentication mechanism using **Kubernetes Secrets**, please configure the authentication mechanism with **Kubernetes Secrets**.
+1. A running Kubernetes cluster with [knative-gcp](../../docs/install/install-knative-gcp.md) installed and configured.
+1. [Pub/Sub Enabled Service Account](../../docs/install/pubsub-service-account.md) installed.
+1. [GCP Broker Deployment](../../docs/install/install-gcp-broker.md#deployment) and [GCP Broker Authentication Setup](../../docs/install/install-gcp-broker.md#authentication-setup-for-gcp-broker).
+1. [Broker with Pub/Sub Channel](../../docs/install/install-broker-with-pubsub-channel.md) installed.
+1. [CloudStorageSource Prerequisites](../../docs/examples/cloudstoragesource/README.md#prerequisites).
+   Note that you only need to: 
+   1. Create with an App Engine application in your project.
+1. [CloudSchedulerSource Prerequisites](../../docs/examples/cloudschedulersource/README.md#prerequisites).
+   Note that you only need to:
+   1. Enable the Cloud Storage API on your project.
+   1. Give Google Cloud Storage permissions to publish to GCP Pub/Sub.
+1. A docker repo containing [the test images](#test-images). Remember to specify the build tag `e2e`.
```

Comment:
The build tag `e2e` is auto-added by `upload-test-images.sh` so no need to mention it here.

Reply:
It looks like it add tag `latest` insetead of `e2e` https://github.com/knative/eventing/blob/master/test/upload-test-images.sh